src/features/auth/signup/AccountCreated.tsx

```diff
@@ -25,6 +26,10 @@ export function AccountCreated() {
     }
   }
 
+  useEffect(() => {
+    if (user?.id) appsFlyer.logEvent('af_complete_registration', { af_user_id: user?.id })
```

Comment:
```suggestion
    if (user?.id) appsFlyer.logEvent(CampaignEvent.CompleteRegistration, { af_user_id: user.id })
```

serait plus propre non ?  Et `CampaignEvent` tu le mets dans src/libs/campaign/events.ts ?